korma/src/commonMain/kotlin/com/soywiz/korma/geom/Point.kt

```diff
@@ -317,6 +317,17 @@ data class Point(
 
     fun rotate(rotation: Angle, out: Point = Point()): Point =
         out.setToPolar(Angle.between(0.0, 0.0, this.x, this.y) + rotation, this.length)
+
+    fun setLength(length: Double): Point {
```

Comment:
I'm not sure about the name, because set methods usually do not depends on internal state.

I would propose other options:

Allowing to set everything (but defaulting to the internal value), similar to setPolar:
```suggestion
    fun setLength(length: Double, x: Double = this.x, y: double = this.y): Point {
```

```suggestion
    fun setLengthKeepingAngle(length: Double): Point {
```

```suggestion
    fun changeLength(length: Double): Point {
```

Reply:
I believe the `changeLength` option is the best, as it clearly describes the intent of this method.
I will make the adjustments and submit for your review.